libs/auth-api-lib/seeders/data/client-licenses-rls.ts

```diff
@@ -0,0 +1,21 @@
+import { compose, createClient, createScope } from './helpers'
+
+export const up = compose(
+  createClient({
+    clientId: '@rls.is/clients/licenses',
+    clientType: 'machine',
+    displayName: 'RLS digital license machine client',
+    description: 'Enables acccess to the license-api',
+    allowedScopes: [
+      'openid',
+      '@island.is/licenses:firearm',
+      '@island.is/licenses:verify',
+    ],
+  }),
+  createScope({
+    name: '@island.is/licenses:firearm',
+    displayName: 'RLS digital license update scope',
+    description:
+      'Veitir aðgang að uppfærsluvirkni á skírteinum útgefnum af RLS',
+  }),
```

Comment:
Should do this before creating the client which refers to it (move it up).

Reply:
Ah, I just followed the example in the Readme.ts which defines the client first

Reply:
Ah yeah. The readme example works because it first creates the client and when creating the scopes it uses "addToClients" to assign them to the client (which has been created at that point).